### PR TITLE
simulator: cleanup logs.

### PR DESCRIPTION
**Pull request by disksing**
- disable all logs by default. Only print out the simulation result.
- use another logger for simulator.

### CODE REVIEW

File: pkg/faketikv/cluster.go [hunk 1]
```diff
@@ -28,18 +28,20 @@ import (
 // ClusterInfo records all cluster information.
 type ClusterInfo struct {
 	*core.RegionsInfo
-	Nodes map[uint64]*Node
+	Nodes  map[uint64]*Node
+	logger *log.Logger
```
Comment (nolouch) [inline pkg/faketikv/cluster.go]: Can we use a global logger?

Comment (disksing) [reply pkg/faketikv/cluster.go]: Seems we will have dependency problem. Not sure which package to put the global logger in. Any suggestions for that?

Comment (nolouch) [reply pkg/faketikv/cluster.go]: Maybe create a sub-package in faketikv?

**Comment by disksing** [comment]
PTAL @nolouch

File: cmd/simulator/main.go [hunk 2]
```diff
@@ -73,12 +84,22 @@ EXIT:
 	driver.Stop()
 	clean()
 
-	log.Infof("Simulation finish. Conf: %s, TotalTick: %d, Result: %s", *confName, driver.TickCount(), simResult)
+	fmt.Printf("%s [%s] total iteration: %d, time cost: %v\n", simResult, *confName, driver.TickCount(), time.Since(start))
```
Comment (siddontang) [inline cmd/simulator/main.go]: why using fmt here?

Comment (disksing) [reply cmd/simulator/main.go]: It's on purpose. We need to print out the simulation result no matter logs are on or off.

**Pull request merged**